test/helpers.js

```diff
@@ -16,7 +16,7 @@ function getEvaluated(output, modules) {
 				return require("../lib/url/escape");
 			if(module.indexOf("-!/path/css-loader!") === 0)
 				module = module.substr(19);
-			if(modules && modules[module])
+			if(modules && module in modules)
```

Comment:
Why `in`?

Reply:
@evilebottnawi: To allow for falsy modules such as `{ './module': null }` to still be resolved.

Reply:
@kgram thanks for clafiry :+1: